### ISSUE TITLE
Open Source License

### ISSUE THREAD

**Issue by anthonyronda** [Detailing-the-Realm/svnsea2e#45]
Hiya!

Would you consider an open source license?

**Comment by mattraykowski** [comment]
@anthonyronda I haven't seen a response from the original author and while there's a lack of LICENSE file the `package.json` specifies the license as MIT. It also specifies that some code was borrowed from the dnd5e system - on Discord I was informed that if the borrowed code was from the 0.7.x and earlier branch of dnd5e development the project is copyleft into GPLv3, but chances are high that the borrowed code is from 0.8.x and that is licensed MIT.

TL;DR: Without an official ruling from the original author all evidence points to this being licensed under MIT.

**Comment by anthonyronda** [comment]
Thanks for reminding me, I hope to reach out via email next.

It's personally not suitable for the package.json to be the source of truth, although for others it might be. The MIT license specifies "in the software files" and should contain the names of all contributors or a reference to the names of all contributors.

I'm a stickler, basically.

**Comment by psychoph** [comment]
All of the code was developed based on the tutorial and is pre 0.8.x. I followed whatever was in there, which sounds like there wasn't any licensing.  I haven't updated the system to work with 0.8.x. I expected this to be open-source code, seems I just didn't follow the proper process. I hoped one day that Chaosium would take over maintenance or create an official system for FVTT. Let me know

**Comment by mattraykowski** [comment]
I added the MIT license with a license header attributing copyrights to psychoph and myself and updated the package.json to point to that file, if that's OK by psychoph. That change is part of PR #46 .
